pkgs/development/python-modules/azure-mgmt-reservations/default.nix

```
@@ -2,40 +2,40 @@
   lib,
   buildPythonPackage,
   fetchPypi,
+  setuptools,
   msrest,
-  msrestazure,
   azure-common,
   azure-mgmt-core,
-  pythonOlder,
+  typing-extensions,
 }:
 
 buildPythonPackage rec {
   pname = "azure-mgmt-reservations";
   version = "2.3.0";
-  format = "setuptools";
-
-  disabled = pythonOlder "3.6";
+  pyproject = true;
 
   src = fetchPypi {
     inherit pname version;
     extension = "zip";
     hash = "sha256-BHCFEFst5jfyIEo0hm86belpxW7EygZCBJ8PTqzqHKc=";
   };
 
-  propagatedBuildInputs = [
+  build-system = [ setuptools ];
+
+  dependencies = [
     msrest
-    msrestazure
     azure-common
     azure-mgmt-core
+    typing-extensions
   ];
```

```suggestion
  ] ++ lib.optionals (pythonOlder "3.8") [ typing-extensions ];
```

see https://github.com/Azure/azure-sdk-for-python/blob/azure-mgmt-reservations_2.3.0/sdk/reservations/azure-mgmt-reservations/setup.py#L76